doc/examples/with_widgets_overlay/lib/example_game.dart

```diff
@@ -25,14 +23,10 @@ class ExampleGame extends Game with TapDetector {
 
   @override
   void onTap() {
-    if (isPaused) {
+    if (overlays.isActive('PauseMenu')) {
       overlays.remove('PauseMenu');
-      isPaused = false;
     } else {
-      overlays.add(
-        'PauseMenu',
-      );
-      isPaused = true;
```

Comment:
I wonder if a toggle method would be useful as well. might be too specific. I like isActive!